diff of execution of:
echo '(cat | grep in) < infile > outfile; cat outfile' | tr ';' '\n' | [shell]
--- expected	2024-09-10 09:26:56.719582209 +0200
+++ found	2024-09-10 09:26:56.745582208 +0200
@@ -1,7 +1,5 @@
-infile content
-infile second line
 
 ======== stderr =======
-
+[progname]: outfile: No such file or directory
 ======== return =======
-return value: 0
+return value: 1
